.editorconfig

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
```

Comment:
Who should the copyright belong to in general for the broker code? I imagine that Sloan or the AAS would explicitly need to allow the copyright to be held by CERN?